misk-aws/src/main/kotlin/misk/dynamodb/DynamoDbHelpers.kt

```diff
@@ -0,0 +1,40 @@
+package misk.dynamodb
+
+import com.amazonaws.services.dynamodbv2.model.AmazonDynamoDBException
+import java.time.Duration
+import misk.backoff.Backoff
+import misk.backoff.DontRetryException
+import misk.backoff.ExponentialBackoff
+import misk.backoff.retry
+
+object DynamoDbHelpers {
+  val DEFAULT_BACKOFF = ExponentialBackoff(
```

Comment:
Unfortunately this object is [stateful](https://github.com/cashapp/misk/blob/master/misk-core/src/main/kotlin/misk/backoff/ExponentialBackoff.kt#L17). You probably want to instantiate a new one in `dynamoDbRetry`

Reply:
... I really should have seen that coming. My bad

Reply:
nah I think it was just bad API design